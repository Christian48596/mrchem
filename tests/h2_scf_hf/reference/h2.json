{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -0.7
          ],
          "radius": 0.79
        },
        {
          "center": [
            0.0,
            0.0,
            0.7
          ],
          "radius": 0.79
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            -0.7
          ]
        },
        {
          "atom": "h",
          "xyz": [
            0.0,
            0.0,
            0.7
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 7,
      "basis_type": "legendre",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -4
    },
    "printer": {
      "file_name": "h2.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "Hartree-Fock",
        "prec": 0.001,
        "restricted": true,
        "screen": 12.0,
        "type": "sad",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": -1.0,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 0.001,
        "helmholtz_prec": -1.0,
        "kain": 3,
        "light_speed": -1.0,
        "localize": false,
        "max_iter": 5,
        "method": "Hartree-Fock",
        "orbital_thrs": 0.01,
        "proj_prec": 0.001,
        "rotation": 0,
        "shared_memory": false,
        "smooth_prec": 0.001,
        "start_prec": 0.001
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        -1.1189687543466913e-17
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 0.0002532469070147269,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            0.0002532469070147269
          ],
          "vector_el": [
            0.0,
            0.0,
            0.00025324690701661426
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            -0.7
          ]
        },
        {
          "symbol": "H",
          "xyz": [
            0.0,
            0.0,
            0.7
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -0.5959939595571349
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": -1.1919879191142697
      },
      "scf_energy": {
        "E_ee": 1.3121419935296943,
        "E_eext": 0.0,
        "E_el": -1.8480616015467426,
        "E_en": -3.621012427994625,
        "E_kin": 1.1168771440154095,
        "E_next": 0.0,
        "E_nn": 0.7142857142857143,
        "E_nuc": 0.7142857142857143,
        "E_tot": -1.1337758872610282,
        "E_x": -0.6560683110972214,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 1.1787890416647235,
        "E_eext": 0.0,
        "E_el": -1.8126341324334825,
        "E_en": -3.2891862266309877,
        "E_kin": 0.8870825171732135,
        "E_next": 0.0,
        "E_nn": 0.7142857142857143,
        "E_nuc": 0.7142857142857143,
        "E_tot": -1.098348418147768,
        "E_x": -0.589319464640432,
        "E_xc": 0.0,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 1.2655466631130026,
              "E_eext": 0.0,
              "E_el": -1.8437364721059708,
              "E_en": -3.505792520583445,
              "E_kin": 1.0292796767655783,
              "E_next": 0.0,
              "E_nn": 0.7142857142857143,
              "E_nuc": 0.7142857142857143,
              "E_tot": -1.1294507578202566,
              "E_x": -0.6327702914011066,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -1.1294507578202566,
            "energy_update": 0.031102339672488544,
            "mo_residual": 0.08089162095936434,
            "wall_time": 2.661677547
          },
          {
            "energy_terms": {
              "E_ee": 1.3004650236984954,
              "E_eext": 0.0,
              "E_el": -1.8476512159069645,
              "E_en": -3.5919482448787767,
              "E_kin": 1.094061742313159,
              "E_next": 0.0,
              "E_nn": 0.7142857142857143,
              "E_nuc": 0.7142857142857143,
              "E_tot": -1.1333655016212503,
              "E_x": -0.650229737039842,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -1.1333655016212503,
            "energy_update": 0.003914743800993659,
            "mo_residual": 0.027559022927045203,
            "wall_time": 2.837094322
          },
          {
            "energy_terms": {
              "E_ee": 1.3121419935296943,
              "E_eext": 0.0,
              "E_el": -1.8480616015467426,
              "E_en": -3.621012427994625,
              "E_kin": 1.1168771440154095,
              "E_next": 0.0,
              "E_nn": 0.7142857142857143,
              "E_nuc": 0.7142857142857143,
              "E_tot": -1.1337758872610282,
              "E_x": -0.6560683110972214,
              "E_xc": 0.0,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -1.1337758872610282,
            "energy_update": 0.0004103856397779104,
            "mo_residual": 0.008741007047838104,
            "wall_time": 2.482742528
          }
        ],
        "wall_time": 7.981609796
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
