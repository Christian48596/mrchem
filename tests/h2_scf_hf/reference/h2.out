                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         rebase-zora                              ***
***         Git commit hash    faecb83984c1d010ec90-dirty               ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Thu Mar 10 17:22:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (1 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 7
 polynomial type       : Legendre
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700000
    1       H            :        0.000000        0.000000        0.700000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999998867977e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : Hartree-Fock
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  0.887082517173
 E-N energy              :            (au)                 -3.289186226631
 Coulomb energy          :            (au)                  1.178789041665
 Exchange energy         :            (au)                 -0.589319464640
 X-C energy              :            (au)                  0.000000000000
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.714285714286
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.812634132433
 Nuclear energy          :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.098348418148e+00
                         :      (kcal/mol)             -6.892240326489e+02
                         :        (kJ/mol)             -2.883713354475e+03
                         :            (eV)             -2.988758172528e+01
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.611582416170
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.223164832339
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : Hartree-Fock
 Checkpointing           : Off
 Max iterations          : 5
 KAIN solver             : 3
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : Dynamic
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-02
 ZORA potential          : None
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          1.000000e+00          -1.098348418148         -1.098348e+00
    1          8.089162e-02          -1.129450757820         -3.110234e-02
    2          2.755902e-02          -1.133365501621         -3.914744e-03
    3          8.741007e-03          -1.133775887261         -4.103856e-04
---------------------------------------------------------------------------
                      SCF converged in 3 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000       -0.700000
    1       H            :        0.000000        0.000000        0.700000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000       -0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  1.116877144015
 E-N energy              :            (au)                 -3.621012427995
 Coulomb energy          :            (au)                  1.312141993530
 Exchange energy         :            (au)                 -0.656068311097
 X-C energy              :            (au)                  0.000000000000
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  0.714285714286
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -1.848061601547
 Nuclear energy          :            (au)                  0.714285714286
---------------------------------------------------------------------------
 Total energy            :            (au)             -1.133775887261e+00
                         :      (kcal/mol)             -7.114551049802e+02
                         :        (kJ/mol)             -2.976728161169e+03
                         :            (eV)             -3.085161222867e+01
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -0.595993959557
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -1.191987919114
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000        0.000253
 Magnitude               :            (au)                        0.000253
                         :         (Debye)                        0.000644
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000        0.000253
 Magnitude               :            (au)                        0.000253
                         :         (Debye)                        0.000644
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 12s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
